src/modules/auth/header.js

```diff
@@ -80,6 +82,11 @@ function HeaderComponent(props) {
 						</DropdownItem>
```

Comment:
I suppose you should also refactor other dropdown items and wrap them into **Link** component.

Reply:
Same way as you make for `About` droprown item

Reply:
@titkov49
> I suppose you should also refactor other dropdown items and wrap them into **Link** component.

I think only the AccessControlScreen

Reply:
Done 👍